client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/launchpad-site-preview.tsx

```diff
@@ -1,14 +1,15 @@
 import { addQueryArgs } from '@wordpress/url';
 import { useTranslate } from 'i18n-calypso';
+import { DEVICE_TYPE } from 'calypso/../packages/design-picker/src/constants';
 import { Device } from 'calypso/../packages/design-picker/src/types';
 import WebPreview from 'calypso/components/web-preview/component';
 import PreviewToolbar from '../design-setup/preview-toolbar';
 
 const LaunchpadSitePreview = ( { siteSlug }: { siteSlug: string | null } ) => {
 	const translate = useTranslate();
 	const previewUrl = siteSlug ? 'https://' + siteSlug : null;
-	const defaultDevice = 'phone';
-	const devicesToShow: Device[] = [ 'computer', 'phone' ];
+	const defaultDevice = DEVICE_TYPE.COMPUTER;
+	const devicesToShow: Device[] = [ DEVICE_TYPE.COMPUTER, DEVICE_TYPE.PHONE ];
```

Comment:
improve type safety by importing the types instead of hardcoding them

Reply:
We can expand this logic with a function/switch statement when we have to handle more tailored flows.